ocs_ci/ocs/node.py

```diff
@@ -657,15 +657,17 @@ def get_compute_node_names(no_replace=False):
     """
     platform = config.ENV_DATA.get('platform').lower()
     compute_node_objs = get_typed_nodes()
-    if platform == constants.VSPHERE_PLATFORM or constants.AWS_PLATFORM:
+    if platform == (constants.VSPHERE_PLATFORM or constants.AWS_PLATFORM):
```

Comment:
@prsurve I don't think this is doing what is intended here. `constants.VSPHERE_PLATFORM or constants.AWS_PLATFORM` will always return `"vsphere"` as that is a "truthful" string. I think what we want instead is to use `"in"`.

```python
if platform in [constants.VSPHERE_PLATFORM, constants.AWS_PLATFORM]:
```

This applies to the other changed code as well. We want to see if the platform is in a particular list of platforms and this use of `or` is not achieving that.